Use different ports between test cases to support Maven parallel execution

## Feature Request

### Is your feature request related to a problem?

When execute Maven command in parallel, test cases failed.
```bash
mvn clean install -T1C
```

```
[ERROR] 2021-05-26 19:39:11.162 [Thread-1] o.a.c.test.TestingZooKeeperServer - From testing server (random state: false) for instance: InstanceSpec{dataDirectory=target/test_zk_data/121959016302313, port=3181, electionPort=64198, quorumPort=64199, deleteDataDirectoryOnClose=true, serverId=1, tickTime=-1, maxClientCnxns=-1, customProperties={}, hostname=127.0.0.1} org.apache.curator.test.InstanceSpec@59c3c1ea
java.net.BindException: Address already in use
	at sun.nio.ch.Net.bind0(Native Method)
	at sun.nio.ch.Net.bind(Net.java:438)
	at sun.nio.ch.Net.bind(Net.java:430)
	at sun.nio.ch.ServerSocketChannelImpl.bind(ServerSocketChannelImpl.java:225)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:74)
	at sun.nio.ch.ServerSocketAdaptor.bind(ServerSocketAdaptor.java:67)
	at org.apache.zookeeper.server.NIOServerCnxnFactory.configure(NIOServerCnxnFactory.java:676)
	at org.apache.zookeeper.server.ServerCnxnFactory.configure(ServerCnxnFactory.java:109)
	at org.apache.zookeeper.server.ServerCnxnFactory.configure(ServerCnxnFactory.java:105)
	at org.apache.curator.test.TestingZooKeeperMain.internalRunFromConfig(TestingZooKeeperMain.java:248)
	at org.apache.curator.test.TestingZooKeeperMain.runFromConfig(TestingZooKeeperMain.java:132)
	at org.apache.curator.test.TestingZooKeeperServer$1.run(TestingZooKeeperServer.java:158)
	at java.lang.Thread.run(Thread.java:748)

  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::       (v1.5.22.RELEASE)


  .   ____          _            __ _ _
 /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
 \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
  '  |____| .__|_| |_|_| |_\__, | / / / /
 =========|_|==============|___/=/_/_/_/
 :: Spring Boot ::       (v1.5.22.RELEASE)

line 1:7 no viable alternative at input 'CREATESHARDING'
line 1:7 no viable alternative at input 'CREATESHARDING'
line 1:15 no viable alternative at input 'CREATESHARDING'
line 1:15 no viable alternative at input 'CREATESHARDING'
line 1:6 no viable alternative at input 'SELECT'
line 1:6 no viable alternative at input 'SELECT'
line 1:0 no viable alternative at input 'SELECT'
line 1:0 no viable alternative at input 'SELECT'
19:39:53.157 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutorCallback
19:39:53.236 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.parser.ShardingSphereSQLParserEngine
19:39:53.302 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask
19:39:53.284 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutorCallback
19:39:53.360 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.parser.ShardingSphereSQLParserEngine
19:39:53.398 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask
19:39:53.814 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutorCallback
19:39:53.814 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.parser.ShardingSphereSQLParserEngine
19:39:53.814 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask
19:39:53.940 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutorCallback
19:39:53.940 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.parser.ShardingSphereSQLParserEngine
19:39:53.940 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask
19:39:56.288 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutorCallback
19:39:56.289 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.parser.ShardingSphereSQLParserEngine
19:39:56.289 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask
19:39:56.294 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.executor.sql.execute.engine.driver.jdbc.JDBCExecutorCallback
19:39:56.294 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.infra.parser.ShardingSphereSQLParserEngine
19:39:56.294 [main] INFO org.apache.shardingsphere.agent.plugin.tracing.AgentRunner - It is successful to enhance the class org.apache.shardingsphere.proxy.frontend.command.CommandExecutorTask
May 26, 2021 7:39:58 PM zipkin2.reporter.AsyncReporter$BoundedAsyncReporter close
WARNING: Timed out waiting for in-flight spans to send
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 51.147 s <<< FAILURE! - in org.apache.shardingsphere.spring.namespace.governance.GovernanceEncryptNamespaceTest
[ERROR] org.apache.shardingsphere.spring.namespace.governance.GovernanceEncryptNamespaceTest  Time elapsed: 51.147 s  <<< ERROR!
java.lang.RuntimeException: org.apache.curator.test.FailedServerStartException: Timed out waiting for server startup
	at org.apache.shardingsphere.spring.namespace.governance.GovernanceEncryptNamespaceTest.init(GovernanceEncryptNamespaceTest.java:48)
Caused by: org.apache.curator.test.FailedServerStartException: Timed out waiting for server startup
	at org.apache.shardingsphere.spring.namespace.governance.GovernanceEncryptNamespaceTest.init(GovernanceEncryptNamespaceTest.java:48)

[ERROR] Errors: 
[ERROR]   GovernanceEncryptNamespaceTest.init:48 » Runtime org.apache.curator.test.Faile...
[ERROR] Tests run: 18, Failures: 0, Errors: 1, Skipped: 1
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.0:test (default-test) on project shardingsphere-jdbc-governance-spring-namespace: There are test failures.
[ERROR] 
[ERROR] Please refer to /Users/wuweijie/IdeaProjects/shardingsphere/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/target/surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date]-jvmRun[N].dump, [date].dumpstream and [date]-jvmRun[N].dumpstream.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :shardingsphere-jdbc-governance-spring-namespace

Process finished with exit code 1
```

### Describe the feature you would like.

Use different ports in different test cases when using embedded ZooKeeper server.

### How?

Make sure ports are different:
* org.apache.shardingsphere.scaling.core.fixture.EmbedTestingServer
* org.apache.shardingsphere.spring.namespace.governance.util.EmbedTestingServer
* org.apache.shardingsphere.spring.boot.governance.util.EmbedTestingServer

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/java/org/apache/shardingsphere/spring/boot/governance/util/EmbedTestingServer.java
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/java/org/apache/shardingsphere/spring/boot/governance/util/EmbedTestingServer.java
@@ -30,7 +30,7 @@ import java.io.IOException;
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public final class EmbedTestingServer {
     
-    private static final int PORT = 3181;
+    private static final int PORT = 3183;
     
     private static volatile TestingServer testingServer;
     

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/resources/application-encrypt.properties
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/resources/application-encrypt.properties
@@ -36,4 +36,4 @@ spring.shardingsphere.governance.name=governance-spring-boot-encrypt-test
 spring.shardingsphere.governance.overwrite=true
 
 spring.shardingsphere.governance.registry-center.type=TestRegistry
-spring.shardingsphere.governance.registry-center.server-lists=localhost:3181
+spring.shardingsphere.governance.registry-center.server-lists=localhost:3183

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/resources/application-readwrite-splitting.properties
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/resources/application-readwrite-splitting.properties
@@ -46,5 +46,5 @@ spring.shardingsphere.governance.name=governance-spring-boot-read-query-test
 spring.shardingsphere.governance.overwrite=true
 
 spring.shardingsphere.governance.registry-center.type=TestRegistry
-spring.shardingsphere.governance.registry-center.server-lists=localhost:3181
+spring.shardingsphere.governance.registry-center.server-lists=localhost:3183
 

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/resources/application-registry-readwrite-splitting.properties
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/resources/application-registry-readwrite-splitting.properties
@@ -19,4 +19,4 @@ spring.shardingsphere.governance.name=governance-spring-boot-registry-readwrite-
 spring.shardingsphere.governance.overwrite=true
 
 spring.shardingsphere.governance.registry-center.type=TestRegistry
-spring.shardingsphere.governance.registry-center.server-lists=localhost:3181
+spring.shardingsphere.governance.registry-center.server-lists=localhost:3183

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/resources/application-registry.properties
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/resources/application-registry.properties
@@ -19,4 +19,4 @@ spring.shardingsphere.governance.name=governance-spring-boot-test
 spring.shardingsphere.governance.overwrite=true
 
 spring.shardingsphere.governance.registry-center.type=TestRegistry
-spring.shardingsphere.governance.registry-center.server-lists=localhost:3181
+spring.shardingsphere.governance.registry-center.server-lists=localhost:3183

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/resources/application-sharding.properties
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-boot-starter/src/test/resources/application-sharding.properties
@@ -71,5 +71,5 @@ spring.shardingsphere.props.executor-size=100
 
 spring.shardingsphere.governance.name=governance-spring-boot-sharding-test
 spring.shardingsphere.governance.registry-center.type=TestRegistry
-spring.shardingsphere.governance.registry-center.server-lists=localhost:3181
+spring.shardingsphere.governance.registry-center.server-lists=localhost:3183
 spring.shardingsphere.governance.overwrite=true

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/src/test/java/org/apache/shardingsphere/spring/namespace/governance/util/EmbedTestingServer.java
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/src/test/java/org/apache/shardingsphere/spring/namespace/governance/util/EmbedTestingServer.java
@@ -30,7 +30,7 @@ import java.io.IOException;
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public final class EmbedTestingServer {
     
-    private static final int PORT = 3181;
+    private static final int PORT = 3182;
     
     private static volatile TestingServer testingServer;
     

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/src/test/resources/META-INF/rdb/encrypt-governance.xml
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/src/test/resources/META-INF/rdb/encrypt-governance.xml
@@ -25,7 +25,7 @@
                            ">
     <import resource="namespace/encrypt-data-source-namespace.xml" />
     
-    <governance:reg-center id="regCenter" type="TestRegistry" server-lists="localhost:3181">
+    <governance:reg-center id="regCenter" type="TestRegistry" server-lists="localhost:3182">
         <props>
             <prop key="max-retries">3</prop>
             <prop key="operation-timeout-milliseconds">1000</prop>

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/src/test/resources/META-INF/rdb/readwrite-splitting-governance.xml
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/src/test/resources/META-INF/rdb/readwrite-splitting-governance.xml
@@ -26,7 +26,7 @@
                            ">
     <import resource="namespace/readwrite-splitting-data-source-namespace.xml" />
     
-    <governance:reg-center id="regCenter" type="TestRegistry" server-lists="localhost:3181">
+    <governance:reg-center id="regCenter" type="TestRegistry" server-lists="localhost:3182">
         <props>
             <prop key="max-retries">3</prop>
             <prop key="operation-timeout-milliseconds">1000</prop>

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/src/test/resources/META-INF/rdb/sharding-governance.xml
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/src/test/resources/META-INF/rdb/sharding-governance.xml
@@ -34,7 +34,7 @@
         <prop key="operation-timeout-milliseconds">1000</prop>
     </util:properties>
     
-    <governance:reg-center id="registryCenterRepository" type="TestRegistry" server-lists="localhost:3181">
+    <governance:reg-center id="registryCenterRepository" type="TestRegistry" server-lists="localhost:3182">
         <props>
             <prop key="max-retries">3</prop>
             <prop key="operation-timeout-milliseconds">1000</prop>

--- a/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/src/test/resources/META-INF/rdb/sharding-readwrite-splitting-governance.xml
+++ b/shardingsphere-jdbc/shardingsphere-jdbc-spring/shardingsphere-jdbc-governance-spring/shardingsphere-jdbc-governance-spring-namespace/src/test/resources/META-INF/rdb/sharding-readwrite-splitting-governance.xml
@@ -26,7 +26,7 @@
                            ">
     <import resource="namespace/sharding-readwrite-splitting-namespace.xml" />
     
-    <governance:reg-center id="regCenter" type="TestRegistry" server-lists="localhost:3181">
+    <governance:reg-center id="regCenter" type="TestRegistry" server-lists="localhost:3182">
         <props>
             <prop key="max-retries">3</prop>
             <prop key="operation-timeout-milliseconds">1000</prop>

